.github/config/android-arm.json

```diff
@@ -9,7 +9,6 @@
       "3",
       "black_box_test"
     ],
```

Comment:
The trailing comma has to be removed. JSON doesn't allow it

Reply:
Done. (JSON sucks!)